service/src/test/java/uk/gov/hmcts/reform/fpl/service/MessageJudgeServiceTest.java

```diff
@@ -495,7 +524,7 @@ void shouldUpdateExistingJudicialMessageWhenReplying() {
                 .updatedTime(time.now())
                 .status(OPEN)
                 .latestMessage(messageReply)
-                .messageHistory(formattedMessageHistory)
+                .messageHistory(String.format("%s - %s", MESSAGE_SENDER, formattedMessageHistory))
```

Comment:
is that in the acceptance criteria or just forgotten?

Reply:
It already exists. the test data was incorrect. I should have updated the `formattedMessageHistory`. I'll do it now